gdmix-workflow/images/gdmix/Dockerfile

```diff
@@ -35,14 +35,14 @@ RUN rm -rf /usr/bin/python && \
 RUN sed -i 's/python/python2/g' /usr/bin/yum
 RUN sed -i 's/python/python2/g' /usr/libexec/urlgrabber-ext-down
 
-# Install spark 2.4.6 and its dependencies
+# Install spark 2.4.7 and its dependencies
 RUN yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && \
     mkdir -p /opt/spark && \
     touch /opt/spark/RELEASE && \
     echo "auth required pam_wheel.so use_uid" >> /etc/pam.d/su && \
     chgrp root /etc/passwd && chmod ug+rw /etc/passwd
 
-ARG spark_version=2.4.6
+ARG spark_version=2.4.7
```

Comment:
Why do we have to use 2.4.7 instead of 2.4.7?

Reply:
Because the 2.4.6 is not downloadable any more, 2.4.7 has been tested and is compatible.